app/components/story-list/template.hbs

```diff
@@ -1,7 +1,8 @@
-<ul class="list list--noborder">
+<ul class="list list--borderbottom">
```

Comment:
yes, it's going to update the other template. Wanted! Less differences, better, no? 😃